numpy/linalg/_linalg.py

```diff
@@ -116,7 +116,7 @@ def _raise_linalgerror_qr(err, flag):
 
 def _makearray(a):
     new = asarray(a)
-    wrap = getattr(a, "__array_prepare__", new.__array_wrap__)
+    wrap = getattr(a, "__array_wrap__", new.__array_wrap__)
```

Comment:
So, if anyone can tell me why this was changed to use array-prepare rather than array-wrap at some point, then...

Reply:
I always wondered about that one too...